packages/genui/a2ui-playground/src/styles.css

```
@@ -727,6 +727,313 @@ a {
+  text-align: left;
+  transition:
+    transform var(--geist-transition),
+    border-color var(--geist-transition),
+    box-shadow var(--geist-transition),
+    background var(--geist-transition);
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  margin: 0 0 14px;
+  break-inside: avoid;
+  page-break-inside: avoid;
```

Stylelint (17.10.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)